pom.xml

```diff
@@ -48,7 +48,7 @@
     <sonar.projectKey>${sonar.organization}_cwa-${project.artifactId}</sonar.projectKey>
 
     <!-- https://spring.io/projects/spring-boot -->
-    <spring-boot.version>2.4.4</spring-boot.version>
+    <spring-boot.version>2.4.3</spring-boot.version>
```

Comment:
We change the version of spring boot to 2.4.4. Maybe we should keep it.

Reply:
@ioangut - right. Main branch versions are running fine, correct? Then please keep the newer version

Reply:
it should be the same for spring cloud version then @ioangut

Reply:
I checked and Spring Cloud version is the same @FelixRottler